server/lib/nl/common/utterance.py

```diff
@@ -78,6 +79,7 @@ class ChartType(IntEnum):
   SCATTER_CHART = 5
   EVENT_CHART = 6
   RANKED_TIMELINE_COLLECTION = 7
+  ANSWER_WITH_ENTITY_OVERVIEW = 8
```

Comment:
nit: just ENTITY_OVERVIEW

Reply:
hm, would ANSWER_WITH_ENTITY_OVERVIEW be more clear though because it's going to be an answer tile with an optional entity overview tile ..

Reply:
Oh I see, got it!  (add that comment:  "Represents an answer tile with an optional entity overview tile")

Reply:
done